README.rst

```diff
@@ -89,6 +89,8 @@ Install Ansible and the ``origin-ci-tool`` in the virtual environment:
 
     $ pip install git+https://github.com/openshift/origin-ci-tool.git --process-dependency-links
 
+**Note:** If you come across `ImportError: No module named cryptography.hazmat.backends`, run the previous command with the additional option `--ignore-installed`
```

Comment:
Not technically correct -- the `ImportError` can be caused by an older installation being present in your system `site-packages` or from a mis-installation of the `cryptography` package, for instance if you did not do the `yum install` steps and `cryptography` failed mid-install. We should add a section linking to [the `cryptography` doc](https://cryptography.io/en/latest/installation/#building-cryptography-on-linux) for installation and add both solutions to this -- use `--force-reinstall` if it's a locally broken install, and `--ignore-installed` if it's pulling in deps from the system.